crates/sfpe_handbook/src/chapter_50/equation_50_2.rs

```diff
@@ -0,0 +1,29 @@
+pub fn pressure_difference_fire_compartment(t_0: f64, t_f: f64, z: f64) -> f64 {
+    let t_0_abs = t_0 + 273.0;
+    let t_f_abs = t_f + 273.0;
+    3460.0 * (1.0 / (t_0_abs) - 1.0 / (t_f_abs)) * z
```

Comment:
```suggestion
    3460.0 * ((1.0 / t_0_abs) - (1.0 / t_f_abs)) * z
```
The brackets you have just surround a single variable. If we want to make it more readable with brackets, I would do it like that.